src/TokenClient.ts

```diff
@@ -86,9 +86,6 @@ export class TokenClient {
         if (!args.code) {
             logger.throw(new Error("A code is required"));
         }
-        if (!args.code_verifier) {
-            logger.throw(new Error("A code_verifier is required"));
```

Comment:
if you look here 
![image](https://user-images.githubusercontent.com/38865738/219423018-7c0b9742-2158-4c98-9872-a745e23b90e6.png)
it seemed someone wanted to make it optional but didn't go through it
instead threw  an error here.